ibu-imager/seedcreator/seedcreator.go

```diff
@@ -500,19 +495,42 @@ func (s *SeedCreator) waitTillOvnKubeNodeIsDown(ctx context.Context) {
 	defer deadlineCancel()
 }
 
-func (s *SeedCreator) getCatalogImages(ctx context.Context) ([]string, error) {
+// filterCatalogImages filters catalog source images as catalog sources have pull always policy
+// and there is no point to precache them.
+// List of catalog images are taken from catalog sources that are currently configured on cluster and known ocp ones.
+// Filtering known images will allow us to fix the case where those images were pulled in seed
+// and after it their catalog sources were removed but images are still part of podman images list as we create seed
+func (s *SeedCreator) filterCatalogImages(ctx context.Context, images []string) ([]string, error) {
+	defaultCatalogsImages := []string{"registry.redhat.io/redhat/community-operator-index",
```

Comment:
Why do we need the defaultCatalogImages + get the catalogSources list from the API?
Why not filter out any image that starts with one of these `registry...-index` ?

Reply:
https://github.com/openshift-kni/lifecycle-agent/pull/220/files#diff-af44b48262b65ed56f5f9d016b4adb3fa8f72e0313e666d5a325b97426b718a8R501